axelrod/strategies/grudger.py

```diff
@@ -63,16 +63,16 @@ def __init__(self) -> None:
     def strategy(self, opponent: Player) -> Action:
         """Begins by playing C, then plays D for mem_length rounds if the
         opponent ever plays D."""
-        if self.grudge_memory >= self.mem_length:
```

Comment:
strategy docstring says defect for 10 rounds.  strategy code said detect for 11 rounds.  I changed the code to fit the docstring.